english/marshmallow.md

```diff
@@ -562,3 +562,5 @@ and as you do a large pink marshmallow shoots out of your hand into the wall bef
 [Explore your new marshmallow powers.](marshmallow-wizard/marshmallow-wizard.md)
 
 [The evil and heroic marshmallow](the-slayer/marshmallow-slayer.md)
+
+All in a hard days work, do you play some [music](play-music/playmusic.md), or do you do the [robot](happy-robot/happy-robot.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.